### PR TITLE
Change SDN interface address to city.

### PR DESCRIPTION
**Pull request by vkaracic**
Since we are using the city as an address parameter in the SDN check, it makes more sense semantically the parameter ``city`` throughout our code except for when making the call to the Treasury endpoint.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vkaracic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=1223"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

File: ecommerce/extensions/payment/migrations/0017_auto_20170328_1445.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('payment', '0016_auto_20170227_1402'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```
Comment (clintonb) [inline ecommerce/extensions/payment/migrations/0017_auto_20170328_1445.py]: You will need to manually call RenameField.

**Review by clintonb — CHANGES_REQUESTED** [review]

**Comment by vkaracic** [comment]
@clintonb comment addressed!

File: ecommerce/extensions/payment/migrations/0017_auto_20170328_1445.py [hunk 2]
```diff
@@ -12,11 +12,12 @@ class Migration(migrations.Migration):
...
     operations = [
-        migrations.RemoveField(
+        migrations.RenameField(
             model_name='sdncheckfailure',
-            name='address',
+            old_name='address',
+            new_name='city',
         ),
-        migrations.AddField(
+        migrations.AlterField(
```
Comment (vkaracic) [inline ecommerce/extensions/payment/migrations/0017_auto_20170328_1445.py]: Not sure if this part is needed though. The ``city`` length on the form is 32.

**Review by clintonb — APPROVED** [review]
Remember to squash your commits.

**Comment by vkaracic** [comment]
I'll try :smile:

**Review by mjfrey — APPROVED** [review]

File: ecommerce/extensions/payment/utils.py [hunk 3]
```diff
@@ -93,7 +93,7 @@ def search(self, name, address, country):
             'api_key': self.api_key,
             'type': 'individual',
             'name': unicode(name).encode('utf-8'),
-            'address': unicode(address).encode('utf-8'),
+            'address': unicode(city).encode('utf-8'),
```
Comment (davec-edx) [inline ecommerce/extensions/payment/utils.py]: add comment to explain why they 'address' key contains the city.  I know we've talked about this, but who will remember in 6 months to fill the next person in on it?

**Comment by davec-edx** [comment]
I'd like to see a comment in the view to inform a future dev why it's address=city (commented on the exact line) but otherwise, looks good.

**Pull request merged**